dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentAsDraftActionlet.java

```diff
@@ -29,16 +31,27 @@
 public class SaveContentAsDraftActionlet extends WorkFlowActionlet {
 
 	private final ContentletAPI   contentletAPI;
-	private final RelationshipAPI relationshipAPI;
 	private final CategoryAPI     categoryAPI;
 	private final PermissionAPI   permissionAPI;
 
 	public SaveContentAsDraftActionlet() {
+		this(
+			APILocator.getContentletAPI(),
+			APILocator.getCategoryAPI(),
+			APILocator.getPermissionAPI()
+		);
+	}
+
+	@VisibleForTesting
+	public SaveContentAsDraftActionlet(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337055)